webui/react/src/e2e/tests/experimentList.spec.ts

```diff
@@ -677,5 +680,45 @@ test.describe('Experiment List', () => {
       );
       await expect(newProjectRows.length).toBe(1);
     });
+
+    test('kill experiment', async ({
+      newProject: {
+        response: { project },
+      },
+      newWorkspace: {
+        response: { workspace },
+      },
+    }) => {
+      killExperiment = Number(
+        detExecSync(
+          `experiment create ${fullPath('examples/tutorials/core_api/2_checkpoints.yaml')} --project_id ${project.id}`,
+        ).split(' ')[2],
```

Comment:
i'm noticing that the CI resources are higher with this PR than without. could you try starting this experiment paused?
```suggestion
        detExecSync(
          `experiment create ${fullPath('examples/tutorials/core_api/2_checkpoints.yaml')} --project_id ${project.id} --paused`,
        ).split(' ')[2],
```